packages/ui/src/machines/authenticator/defaultServices.ts

```diff
@@ -60,6 +66,68 @@ export const defaultServices = {
 
   // Validation hooks for overriding
   async validateCustomSignUp(formData, touchData): Promise<ValidatorResult> {},
+  async validateFormPassword<Validator>(
+    formData,
+    touchData,
+    passwordSettings: PasswordSettings
+  ): Promise<ValidationComplexityError> {
+    const { password } = formData;
+
+    const { password: touched_password } = touchData;
+
+    /**
+     * If the password is not touched,
+     * or if the password settings are not set, we don't need to validate it.
+     */
+    if (!touched_password || !passwordSettings) return null;
+
+    const password_complexity = [];
+
+    const policyMinLength = +passwordSettings?.passwordPolicyMinLength;
+    if (password.length < policyMinLength) {
+      password_complexity.push(
+        `Password must contain at least ${policyMinLength} characters`
+      );
+    }
+
+    const passwordPolicyCharacters = passwordSettings?.passwordPolicyCharacters;
+
+    passwordPolicyCharacters?.forEach((errorCheck) => {
+      switch (errorCheck) {
+        case 'REQUIRES_LOWERCASE':
+          if (!/[a-z]/.test(password))
+            password_complexity.push(
+              `Password must contain a lower case letter`
```

Comment:
The sentence is long already, and on smaller screen sizes, I'm worried we'll have to set the font-size to be even smaller. So I think this is fine.

Reply:
+1 on using quotes. Not sure why I used template literals.

Reply:
Updated quotes in 11fba8aca5ddf4546428a9d54b9f30d0cffa0ba8

Reply:
Hmmm, concerned that we are implying that users should only include a **single** lowercase/special/uppercase/etc character in their password. Also, with i18n we cannot guarantee the rendering length, correct?

Reply:
What if we changed it to `Password must have lowercase characters`. This would help suggest that it could contain one or more characters. This is also apart of the message Cognito sends back for this error

<img width="1862" alt="image" src="https://user-images.githubusercontent.com/65630/157512058-ecb57721-b3a8-4db2-806e-eed65e64a276.png">

Reply:
Think that works for sure!

Reply:
OK I updated it here 52cf527f4e43993655ff79f7646debd5a120c33f